superdesk/sams/decorator.py

```diff
@@ -0,0 +1,14 @@
+from functools import wraps
+from flask import request
```

Comment:
You can also use add `current_app as app` here so the decorator doesn't require the app to be passed in

Reply:
Added